### PR TITLE
[FEAT] INT-565: Add initial UseCase support

### PR DESCRIPTION
**Pull request by thesteady**
## Problem
We want to provide new "use cases" in Console to help folks find the best workflow for their jobs. We want to be able to make quick changes to the available use cases, including associated thumbnail images, without needing to redeploy Console.

## Solution
Define the use case card configs here in gretel-blueprints repo, which we can then query from the Console. This also adds the initial set of images we plan to use. I added a test suite around the use cases, including using JSON Schema to validate that each card conforms to a data shape we are expecting (e.g., required fields, enum values). In a followup PR I plan to add a GitHub Action to run these automatically in CI. _Misc: updated repo README to include new directory info._

## Notes
Initial use cases / text is likely to change still, these are placeholder values that will let me keep developing and wire everything together. The great thing is that the way we're designing this makes it easy to update these!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Gretel Blueprints provide example Gretel Configurations, Notebooks, and other utilities for solving specific use cases and data challenges.
 
-Blueprint contents and any supporting modules within this repository are licensed under Apache 2.0.  Many Blueprints also will install our [gretel-client](https://github.com/gretelai/gretel-python-client) and [gretel-synthetics](https://github.com/gretelai/gretel-synthetics) open source packages.
+Blueprint contents and any supporting modules within this repository are licensed under Apache 2.0. Many Blueprints also will install our [gretel-client](https://github.com/gretelai/gretel-python-client) and [gretel-synthetics](https://github.com/gretelai/gretel-synthetics) open source packages.
```
Comment (yamini) [inline README.md:5]: > open source packages

Should this be SAL or Software Available Licensed packages?

Comment (thesteady) [reply README.md:5]: I talked to John and he said this one is fine as is.

File: test_use_cases.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+def validate_images_exist(image_name):
+    dir_path = "use_cases/images"
+    split_name = image_name.split(".")
+    two_x = split_name[0] + "@2x."+  split_name[1]
+    three_x = split_name[0] + "@3x."+  split_name[1]
+  
+    assert os.path.exists((Path(__file__).parent / dir_path / image_name))
+    assert os.path.exists((Path(__file__).parent / dir_path / two_x))
+    assert os.path.exists((Path(__file__).parent / dir_path / three_x))
```
Comment (johntmyers) [inline test_use_cases.py:95]: suggestion: The `Path` object has its own `is_file()` method, so you can do something like assert `Path(...).is_file()` as well

Comment (thesteady) [reply test_use_cases.py:95]: ah, nice! will update.

**Review by johntmyers — APPROVED** [review]

**Pull request merged**